Sources/SwiftKuery/Delete.swift

```diff
@@ -28,6 +28,9 @@ public struct Delete: Query {
     /// A String with a clause to be appended to the end of the query.
     public private (set) var suffix: QuerySuffixProtocol?
     
+    /// An array of `AuxiliaryTable` which will be used in WITH clause.
```

Comment:
Please change to: An array of `AuxiliaryTable` which will be used in **a query with a** WITH clause.

Please do this in Insert, Select, and Update as well